<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studenteditquestions</class>
 <widget class="QWidget" name="studenteditquestions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Your Profile</string>
  </property>
  <widget class="QSpinBox" name="spinBox_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_6">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>190</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_5">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_7">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>240</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_17">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>120</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_12">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>470</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_21">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>290</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_19">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>210</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_22">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>320</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_16">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>100</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_13">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>40</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_20">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>240</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_24">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>470</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_14">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>60</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_15">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>80</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_18">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>190</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PERSONAL QUESTIONS </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>How well did you do in 2401?  </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>301</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>How well did you do in 2404?  </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>How well did you do in 1804?  </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>How punctual are you ?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>how likely are you resolve a conflict in the group ?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Are you comfortable in expressing your idea?</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_10">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>380</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_9">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>330</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_23">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>410</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_8">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>300</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_11">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>430</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Would you facilitate a struggling team member ?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string> Do you make decisions by your instincts or by analysis ?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>How logical are you ?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>321</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>How good are you when dealing with critisism ?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>421</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>How comfortable are you presenting infornt of an audience ?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>470</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose your preffered language ?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>500</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>1. C++           2. Java           3. Python</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>20</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PARTNER QUESTIONS </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>321</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>How well did you do in 2401?  </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>70</y>
     <width>321</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>How well did you do in 2404?  </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>90</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>How well did you do in 1804?  </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>110</y>
     <width>281</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>How punctual are you ?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>130</y>
     <width>371</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>how likely are you resolve a conflict in the group ?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>190</y>
     <width>331</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Are you comfortable in expressing your idea?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>341</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Would you facilitate a struggling team member ?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>240</y>
     <width>391</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string> Do you make decisions by your instincts or by analysis ?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>290</y>
     <width>351</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>How logical are you ?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>330</y>
     <width>381</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>How good are you when dealing with critisism ?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>476</x>
     <y>370</y>
     <width>421</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>How comfortable are you presenting infornt of an audience ?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>440</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose your preffered language ?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>470</y>
     <width>281</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>1. C++           2. Java           3. Python</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1001</width>
     <height>581</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_29</zorder>
  <zorder>spinBox_3</zorder>
  <zorder>spinBox_6</zorder>
  <zorder>spinBox_5</zorder>
  <zorder>pushButton</zorder>
  <zorder>spinBox_7</zorder>
  <zorder>spinBox_2</zorder>
  <zorder>spinBox_4</zorder>
  <zorder>spinBox</zorder>
  <zorder>spinBox_17</zorder>
  <zorder>spinBox_12</zorder>
  <zorder>spinBox_21</zorder>
  <zorder>spinBox_19</zorder>
  <zorder>spinBox_22</zorder>
  <zorder>spinBox_16</zorder>
  <zorder>spinBox_13</zorder>
  <zorder>spinBox_20</zorder>
  <zorder>spinBox_24</zorder>
  <zorder>spinBox_14</zorder>
  <zorder>spinBox_15</zorder>
  <zorder>spinBox_18</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>spinBox_10</zorder>
  <zorder>spinBox_9</zorder>
  <zorder>spinBox_23</zorder>
  <zorder>spinBox_8</zorder>
  <zorder>spinBox_11</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>label_14</zorder>
  <zorder>label_15</zorder>
  <zorder>label_16</zorder>
  <zorder>label_17</zorder>
  <zorder>label_18</zorder>
  <zorder>label_19</zorder>
  <zorder>label_20</zorder>
  <zorder>label_21</zorder>
  <zorder>label_22</zorder>
  <zorder>label_23</zorder>
  <zorder>label_24</zorder>
  <zorder>label_25</zorder>
  <zorder>label_26</zorder>
  <zorder>label_27</zorder>
  <zorder>label_28</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
